be/src/common/config.h

```diff
@@ -662,6 +662,8 @@ CONF_Bool(rewrite_partial_segment, "true");
 CONF_String(object_storage_access_key_id, "");
 CONF_String(object_storage_secret_access_key, "");
 CONF_String(object_storage_endpoint, "");
+// Tencent cos needs to add region information
```

Comment:
For oss we can also set this manually. how about changing the annotation to cover more cases?
and for aws ec2, if we set this then the AWS_REGION or region get by ec2 meta client can not take effect

Reply:
If anyone update the AWS_REGION env, just doen't set the region in be.conf? what do you think about it.

Reply:
LGTM @stephen-shelby what i think about is that we need to mention this risk in the configuration instructions.

Reply:
ok, I will explain it in the documentation about this config.